data/hai/unfettered jobs.txt

```diff
@@ -26,6 +26,7 @@ mission "Unfettered Tribute 1"
 	name "Hai Tribute to <planet>"
 	job
 	repeat
+	clearance "After speaking with the Unfettered and explaining your presence, they give you permission to land with the tribute from the other Hai."
```

Comment:
Maybe replace each set of identical messages with a `phrase` if that's possible?

Reply:
It's not.